python/ray/tune/tests/test_experiment_analysis.py

```diff
@@ -10,10 +10,67 @@
 import pandas as pd
 
 import ray
-from ray.tune import run, sample_from, Analysis
+from ray.tune import run, Trainable, sample_from, Analysis, grid_search
 from ray.tune.examples.async_hyperband_example import MyTrainableClass
 
 
+class ExperimentAnalysisInMemorySuite(unittest.TestCase):
+    class MockTrainable(Trainable):
+        def _setup(self, config):
+            self.id = config["id"]
+            self.idx = 0
+            self.scores_dict = {
```

Comment:
pass in the list through config and use that (index w/ `self.scores[self.idx * 5 + self.id] or smth)